src/main/java/com/google/sps/servlets/IntervieweeFeedbackServlet.java

```diff
@@ -46,18 +47,27 @@
 public class IntervieweeFeedbackServlet extends HttpServlet {
   private ScheduledInterviewDao scheduledInterviewDao;
   private PersonDao personDao;
+  private EmailSender emailSender;
   private Path emailsPath =
       Paths.get(
           System.getProperty("user.home") + "/InterviewMe/src/main/resources/templates/email");
 
   @Override
   public void init() {
-    init(new DatastoreScheduledInterviewDao(), new DatastorePersonDao());
+    EmailSender emailer;
+    try {
+      emailer = new SendgridEmailSender(new Email("interviewme.business@gmail.com"));
```

Comment:
Nit: make this email a final static 
same for InterviewerFeedbackServlet

Reply:
Done